.github/workflows/regression_tests.yml

```diff
@@ -0,0 +1,117 @@
+name: Containerized Regression tests
+
+on: [push, pull_request]
+
+jobs:
+  fastmri_jax:
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run containerized workload
+      run: |
+        docker run  -v /home/kasimbeg/data/:/data/ -v /home/kasimbeg/experiment_runs/:/experiment_runs -v /home/kasimbeg/experiment_runs/logs:/logs --gpus all --ipc=host us-central1-docker.pkg.dev/training-algorithms-external/mlcommons-docker-repo/algoperf_jax:dev -d fastmri -f jax -s baselines/adamw/jax/submission.py -w fastmri -t baselines/adamw/tuning_search_space.json -e test_today/adamw -m 10 -c False -o True -r false
```

Comment:
no big deal if not, but is it possible to pull `/home/kasimbeg` out into a variable?

Reply:
Yes, that is a good suggestion. I can just make it $HOME for now?

Reply:
sgtm